pkg/configv2/default.go

```diff
@@ -0,0 +1,92 @@
+package configv2
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/bacalhau-project/bacalhau/pkg/configv2/types"
+)
+
+var Default = types.Bacalhau{
+	Repo: "~/.bacalhau",
+	Name: "",
+	Client: types.Client{
+		Address: "http://0.0.0.0:1234",
+	},
+	Server: types.Server{
+		Address: "0.0.0.0",
+		Port:    1234,
+		TLS:     types.TLS{},
+		Auth: types.AuthConfig{
+			Methods: map[string]types.AuthenticatorConfig{
+				"ClientKey": {
+					Type: "challenge",
+				},
+			},
+		},
+	},
+	Orchestrator: types.Orchestrator{
+		Enabled:   true,
+		Listen:    "0.0.0.0",
+		Port:      4222,
+		Advertise: "0.0.0.0",
+		NodeManager: types.NodeManager{
+			DisconnectTimeout: types.Duration(time.Second * 30),
+			AutoApprove:       true,
```

Comment:
We can modify this value to be false in the config by omitting it here. 
This would lead to the behavior I'd expect as a user: compute nodes joining a cluster by default require approval. Iff I set this config value to true then they are approved automatically. 

Negating the name of this value to be `ManualApproval` means that an empty field for this config implies compute nodes are automatically approved when joining, which isn't the default behavior we want.